hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestFederationClientInterceptor.java

```diff
@@ -364,7 +362,7 @@ public void testForceKillApplicationNotExists()
       Assert.fail();
     } catch (YarnException e) {
       Assert.assertTrue(e.getMessage().equals(
```

Comment:
assertEquals

Reply:
Thanks for helping to review the code, I will fix it.